Traceback (most recent call last):
  File "C:\Users\ignac\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\ignac\anaconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\ignac\anaconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\ignac\anaconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\ignac\anaconda3\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\ignac\anaconda3\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\ignac\anaconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\ignac\anaconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from matplotlib import rcParams, cycler
import matplotlib.pyplot as plt
import numpy as np
plt.ion()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[1], line 1[0m
[1;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m [38;5;28;01mimport[39;00m rcParams, cycler
[0;32m      2[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpyplot[39;00m [38;5;28;01mas[39;00m [38;5;21;01mplt[39;00m
[0;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mnumpy[39;00m [38;5;28;01mas[39;00m [38;5;21;01mnp[39;00m

File [1;32m~\anaconda3\lib\site-packages\matplotlib\__init__.py:107[0m
[0;32m    103[0m [38;5;28;01mimport[39;00m [38;5;21;01mwarnings[39;00m
[0;32m    105[0m [38;5;66;03m# cbook must import matplotlib only within function[39;00m
[0;32m    106[0m [38;5;66;03m# definitions, so it is safe to import from it here.[39;00m
[1;32m--> 107[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m [38;5;28;01mimport[39;00m _api, cbook, docstring, rcsetup
[0;32m    108[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcbook[39;00m [38;5;28;01mimport[39;00m MatplotlibDeprecationWarning, sanitize_sequence
[0;32m    109[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcbook[39;00m [38;5;28;01mimport[39;00m mplDeprecation  [38;5;66;03m# deprecated[39;00m

File [1;32m~\anaconda3\lib\site-packages\matplotlib\rcsetup.py:27[0m
[0;32m     25[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcbook[39;00m [38;5;28;01mimport[39;00m ls_mapper
[0;32m     26[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcolors[39;00m [38;5;28;01mimport[39;00m Colormap, is_color_like
[1;32m---> 27[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mfontconfig_pattern[39;00m [38;5;28;01mimport[39;00m parse_fontconfig_pattern
[0;32m     28[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01m_enums[39;00m [38;5;28;01mimport[39;00m JoinStyle, CapStyle
[0;32m     30[0m [38;5;66;03m# Don't let the original cycler collide with our validating cycler[39;00m

File [1;32m~\anaconda3\lib\site-packages\matplotlib\fontconfig_pattern.py:15[0m
[0;32m     13[0m [38;5;28;01mimport[39;00m [38;5;21;01mre[39;00m
[0;32m     14[0m [38;5;28;01mimport[39;00m [38;5;21;01mnumpy[39;00m [38;5;28;01mas[39;00m [38;5;21;01mnp[39;00m
[1;32m---> 15[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpyparsing[39;00m [38;5;28;01mimport[39;00m (Literal, ZeroOrMore, Optional, Regex, StringEnd,
[0;32m     16[0m                        ParseException, Suppress)
[0;32m     18[0m family_punc [38;5;241m=[39m [38;5;124mr[39m[38;5;124m'[39m[38;5;130;01m\\[39;00m[38;5;124m\[39m[38;5;124m-:,[39m[38;5;124m'[39m
[0;32m     19[0m family_unescape [38;5;241m=[39m re[38;5;241m.[39mcompile([38;5;124mr[39m[38;5;124m'[39m[38;5;130;01m\\[39;00m[38;5;124m([[39m[38;5;132;01m%s[39;00m[38;5;124m])[39m[38;5;124m'[39m [38;5;241m%[39m family_punc)[38;5;241m.[39msub

[1;31mModuleNotFoundError[0m: No module named 'pyparsing'
ModuleNotFoundError: No module named 'pyparsing'

